src/frontend/components/Chat/SharedChatComponents/ChatInputBar/index.tsx

```diff
@@ -0,0 +1,90 @@
+import { SendButtonSVG } from 'components/SVGIcons/SendButtonSVG';
+import { Box, Button, TextInput } from 'grommet';
+import React, { useState } from 'react';
+import { defineMessages, useIntl } from 'react-intl';
+import styled from 'styled-components';
+import { converse } from '../../../../utils/window';
+
+const messages = defineMessages({
+  inputBarPlaceholder: {
+    defaultMessage: 'Message...',
+    description:
+      'The input bar to write, edit and send messages to the chat conversation.',
+    id: 'components.InputBar.inputBarPlaceholder',
+  },
+});
+
+const SendButtonStyled = styled(Button)`
+  width: 40px;
+  height: 40px;
+  border: 1px solid;
+  border-color: #035ccd;
+  border-radius: 6px;
+  margin: 5px;
+`;
+
+const TextInputStyled = styled(TextInput)`
+  font-weight: normal;
+  color: #035ccd;
+`;
+
+export const ChatInputBar = () => {
+  const [chatMessage, setChatMessage] = useState('');
+  const intl = useIntl();
+
+  const handleSendMessage = async () => {
+    if (chatMessage.length !== 0) {
+      await converse.sendMessageWithConverse(chatMessage);
+      setChatMessage('');
+    }
+  };
+
+  const handleKeypress = (event: React.KeyboardEvent<HTMLInputElement>) => {
+    if (event.code === 'Enter') {
+      handleSendMessage();
```

Comment:
Should be awaited ?

Reply:
I'm not sure to see what difference it will make if awaited, because handleSendMessage is a void function and no instructions are executed after it is called. Maybe I'm missing smthg ?